Webex Connect Flows/v3.0/Sample/Usage of Set Variable In Flows/README.md

```diff
@@ -0,0 +1,49 @@
+
+
+# Set Variable Sample Flows Overview :
+- This example demonstrates the usage of Set Variable node using Live Chat Flow and Task Routed Flow. However, the use case is not limited to the livechat only. Any workflow can use the Set Variable node.
+
+## Set Variable
+- The example uses fields from a pre-chat form to inject into the system, however Set Variable Node can inject any variable into the system.
+- For more details on Set Variable please refer to https://help.imiconnect.io/docs/set-variable. 
+
+The folder includes the following sample flows :
+## Media Specific Workflow :
+- ### Live Chat Inbound Sample Flow with Set Variable :
+    - Every inbound customer message sent over the configured customer chat widget will trigger this workflow.
+    - This workflow represents a typical use case for a bank, where a customer reaches out to customer support executives for issues with credit cards, debit cards, savings accounts, or loan enquiry.
+    - The customer selects the issue type from the pre-chat form dropdown.
+    - The customer can also provide a short description of the issue.
+    - These additional contextual details about the interaction are injected into the contact centre using the "Set Variable" Node. The sample flow gives an example of how to do that using the "Set Flow Variable" method in the "Set Variable" Node.
+    - The Agent can view the injected details in the interaction panel of Agent Desktop.
+  
+      <img width="800" alt="Screen Pop" src="../../images/SetVariable.png">
+
+## Media Agnostic Event Workflow :
+- ### Task Routed Workflow :
+    - The contextual information injected from the media-specific workflows is also available to the shared flow.
+    - An "Evaluate" node can extract these details. The sample flow gives an example of how to do that.
+   
+      <img width="800" alt="Helper method" src="../../images/Helper method.png">
+
+    - In this sample flow we can see that in the Live Chat Inbound Flow we have set the IssueDescription and IssueType.And we have marked these fields as agent viewable so agent will be able to view these variables. Also they are marked as agent editable so agent can also edit these variables. We are getting the values of IssueDescription
+    and IssueType from the customer entered values of IssueDescription and IssueType. We are extracting these values in the Task Routed Flow. We can see these values
+    in the interaction panel on the agent desktop.
+
+      <img width="800" alt="Helper method" src="../../images/chat.png">
+  
+      <img width="800" alt="Helper method" src="../../images/SetVariableExample.png">
+
+
+## Quick Start on Workflows
```

Comment:
Quick Start on Workflows doesnt seem to be apt heading. If its needed we should give something appropriate.